test/js/source/source_cache.test.js

```diff
@@ -722,7 +722,6 @@ test('SourceCache#findLoadedParent', (t) => {
         t.equal(sourceCache.findLoadedParent(new TileCoord(2, 0, 0), 0, retain), tile);
         t.deepEqual(retain, expectedRetain);
         t.equal(sourceCache._cache.order.length, 0);
-        t.equal(sourceCache._tiles[tile.coord.id], tile);
 
         t.end();
     });
```

Comment:
Can you add a test that triggers the render raster loop to ensure this behavior does not regress?